kafka_metric/getmessages_kafka_metric.py

```diff
@@ -167,13 +165,25 @@ def process_parse_messages(log_queue, cli_config_vars, if_config_vars, agent_con
                     continue
 
                 metric_key = '{}[{}]'.format(data_field, full_instance)
-                datas.put({
-                    'project': project,
-                    'timestamp': timestamp,
-                    'full_instance': full_instance,
-                    'metric_key': metric_key,
-                    'value': str(data_value)
-                })
+                if isinstance(project, dict):
+                    for key, value in project.items():
+                        datas.put({
+                            'project': value['custom_project'],
+                            'system_name': value['custom_system'],
```

Comment:
It is possible for this field to be blank at this point. We need to handle the case where the custom project mapping is provided, but the system is not included. 

From the comments: 
4567::Customer-Project-2